src/components/Card/index.jsx

```diff
@@ -0,0 +1,38 @@
+import styles from "./styles.module.css";
+import { getTimeAgo, formatDate } from "util/time";
+import noImg from "assets/noImg.png";
+import starImg from "assets/star.png";
+import kebabImg from "assets/kebab.png";
+
+export default function Card({ link }) {
+  const createdAt = "createdAt" in link ? "createdAt" : "created_at";
+  const imageSource = "imageSource" in link ? "imageSource" : "image_source";
+
+  return (
+    <a href={link.url} target="_blank">
+      <div className={styles.link}>
+        <div className={styles["link-cover"]}>
+          <img src={link[imageSource] || noImg} alt="cardCover" />:
+        </div>
+        <button className={styles.likeBtn}>
+          <img src={starImg} alt="likeBtn" />
+        </button>
+        <div className={styles["link-contents"]}>
+          <div className={styles["link-header"]}>
+            <p className={styles["link-update"]}>
+              {getTimeAgo(link[createdAt])}
+            </p>
+            <button className={styles.kebabBtn}>
+              <img src={kebabImg} alt="kebabImg" />
+            </button>
+          </div>
+          <h2 className={styles["link-title"]}>
+            {link.title === null ? "제목없음" : link.title}
```

Comment:
P3:
falsy 값들일 시 default text가 나올 수 있도록 하면 좋겠습니다.

```suggestion
            {!link.title ? "제목없음" : link.title}
```